CMakeLists.txt

```diff
@@ -115,7 +115,7 @@ endforeach()
 
 # List of all target backends to be built by default:
 set(IREE_ALL_TARGET_BACKENDS
-  # TODO(scotttodd): LLVMAOT
+  DYLIB-LLVM-AOT
```

Comment:
could also (future PR?) update some parts of https://github.com/google/iree/blob/main/scripts/update_op_coverage.py and https://github.com/google/iree/blob/main/scripts/update_e2e_coverage.py

(may also need style changes to make the tables accommodate an extra column)

Reply:
https://github.com/google/iree/issues/2673